apps/ohhai-browser/ohhai-browser.yml

```diff
@@ -0,0 +1,7 @@
+name: 'OhHai Browser'
+description: 'A web browser build around differnt types of tabs.'

```

Comment:
I suggest you change the line from 
`description: 'A web browser build around different types of tabs.'`
to
`description: 'Web browser build around different types of tabs.'`

Reply:
I have just fixed this

Reply:
Kool so now Build works, but im not sure when the pull request will be merged